protocol/communication/README.md

```diff
@@ -3,7 +3,10 @@ Communication
 
 A guide for communicating within thoughtbot.
 
-Tickets
+Pivotal Tracker Stories
 -------
 
-* People assign themselves to tickets. When in doubt, do not assign a ticket to someone.
+* Software Engineers join the bi-weekly IPM to discuss the user stories (and chores, and bugs) for the week.
```

Comment:
Bullet points 1 and 2 should be replaced with something like:
```
* Software Engineers are grouped into squads that work toward product themes over the course of a multi-week cycle. Squad leads organize ad hoc check-ins and planning meetings to scope out new user stories and set context for which people are most apt to work on them.
```

Reply:
Love it.